### PR TITLE
Create project-work.md

### PR DESCRIPTION
**Pull request by christyharagan**
Create a new handbook page to track the process for defining, planning, and performing CE project work.

### CODE REVIEW

**Review by amenne — COMMENTED** [review]
I think this link to project work should be within the team culture page, not from the main CE page since that's where more of the tactical details are, including the forward looking vision that references project work. Similarly, I moved working with product to that page.

File: handbook/ce/project-work.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Customer Engineering Project Work
+
+CEs engage in project work to both enhance the go-to-market capabilities of the company, to enhance the customer experience through reusable assets, and to perform other customer centric work that, for example, may involve development of code.
```
Comment (emchap) [inline handbook/ce/project-work.md:3]:
```suggestion
CEs engage in project work to both enhance the go-to-market capabilities of the company, to enhance the customer experience through reusable assets, and to perform other customer-centric work that, for example, may involve development of code, creation of educational assets, or development of new demo flows.
```

File: handbook/ce/project-work.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ - Add a new entry in the CE Project Planning & Review [agenda document](https://docs.google.com/document/d/1xTq5Tj_y9cV9aDIhQXWj2QN9Z-gbWhvtkCVAk4b0LD0/edit?usp=sharing) as follows:
+   - ```Review proposed project [[link to brief]]```
+   - If you need to suggest a new Project Type, append: ```and recommend project type```
+ - At the next CE Project Planning & Review meeting, the proposal will be discussed, and the following needs to be agreed:
+   - Relative priority of the project and approximate timelines
+   - Project owner, if not already assigned
+   - Relevant stakeholders and their involvement
+   - Project contributors, and their roles
+   - Optionally (and time depending) other fields may be discussed with the wider team
+ - The Project Owner updates the status of the project in Asana to "Agreed" and then, once work starts, to "In Progress"
```
Comment (emchap) [inline handbook/ce/project-work.md:41]: Do we want to change the process so that this is just changed in the meeting? One less step, seems like.

**Review by emchap — APPROVED** [review]
Left one suggestion but this looks good.

**Pull request closed without merging**